### PR TITLE
Add Devise developer login strategy

### PR DESCRIPTION
**Pull request by Frost**
- Bump devise version
- Add developer login strategy

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -21,8 +23,8 @@ gem 'less-rails-bootstrap'
 gem 'mysql2', '~> 0.3.10'
 gem 'net-ldap'
 gem 'nokogiri', '~> 1.5.4'
-gem 'oa-core'
-gem 'oa-enterprise'
+# gem 'oa-core'
+# gem 'oa-enterprise'
```
Comment (Koronen) [inline Gemfile]: Dead code.

File: app/controllers/people/omniauth_callbacks_controller.rb [hunk 2]
```diff
@@ -1,12 +1,29 @@
 class People::OmniauthCallbacksController < Devise::OmniauthCallbacksController
+  skip_before_filter :authorize_user!
+
   def new
     redirect_to user_omniauth_authorize_path(:cas)
   end
 
   def cas
-    @person = Person.find_for_cas_oath(env['omniauth.auth'], current_user)
+    @person = Person.find_for_cas_oath(env["omniauth.auth"], current_user)
```
Comment (Koronen) [inline app/controllers/people/omniauth_callbacks_controller.rb]: Rename this method while you're at it? Should probably be called `Person.find_for_cas_oauth`.

Comment (Frost) [inline app/controllers/people/omniauth_callbacks_controller.rb]: Did you miss a negation somewhere in that sentence? I'll rename it to `Person.find_for_oauth`.

File: app/controllers/people/omniauth_callbacks_controller.rb [hunk 3]
```diff
@@ -1,12 +1,29 @@
...
+  def developer
+    unless Cashflow::Application.settings["enable_developer_login_strategy"]
+      render text: "Permission denied", status: :forbidden
+      return
+    end
+
+    @person = Person.find_for_oauth(env["omniauth.auth"], current_user)
     if @person.persisted?
-      flash[:notice] = I18n.t 'devise.omniauth_callbacks.success', kind: 'CAS'
+      flash[:notice] = I18n.t "devise.omniauth_callbacks.success", kind: "CAS"
```
Comment (Koronen) [inline app/controllers/people/omniauth_callbacks_controller.rb]: `kind: "Developer"`?

**Comment by Koronen** [comment]
[Two](https://github.com/datasektionen/cashflow/pull/132#discussion_r23496097) [comments](https://github.com/datasektionen/cashflow/pull/132#discussion_r23497574), otherwise LGTM! :+1:

**Pull request merged**